doc/sphinx/addendum/universe-polymorphism.rst

```diff
@@ -270,6 +270,44 @@ Insufficiently restrictive variance annotations lead to errors:
 
    Fail Polymorphic Cumulative Record bad@{*a} := {p : Type@{a}}.
 
+.. example:: Demonstration of universe variances
+
+   Here is an example demonstating the behavior of various universe variances.
+
+   .. coqtop:: all
+
+      Set Printing Universes.
+      Set Universe Polymorphism.
+      Set Polymorphic Inductive Cumulativity.
+
+      Inductive Invariant @{=u} : Type@{u}.
+      Inductive Covariant @{+u} : Type@{u}.
+      Inductive Irrelevent@{*u} : Type@{u}.
+
+      Section Universes.
+        Universe u v.
+        Constraint u < v.
+
+        (** An invariant universe blocks cumulativity from upper or lower levels. *)
+        Axiom inv_u : Invariant@{u}.
+        Axiom inv_v : Invariant@{v}.
+        Fail    Check (inv_u : Invariant@{v}).
+        Fail    Check (inv_v : Invariant@{u}).
+
+        (** A covariant universe allows cumulativity from a lower level. *)
+        Axiom co_u : Covariant@{u}.
+        Axiom co_v : Covariant@{v}.
+        Succeed Check (co_u : Covariant@{v}).
```

Comment:
```suggestion
        Check (co_u : Covariant@{v}).
```

`Succeed` isn't needed--it just prints another line of noise.  In 3 place.

Reply:
Right but I didn't want the command to have any side effects.

Reply:
I wasn't aware that `Check` had any side effects.  Even so, if they don't affect the example, it's simpler not to have `Succeed`.